pkg/lib/lock.go

```diff
@@ -208,9 +208,11 @@ func formatLuks(device, name, mapper, label, pass string, logger types.KairosLog
 	}
 
 	l.Debug().Msg("discards")
-	out, err = SH(fmt.Sprintf("cryptsetup refresh --persistent --allow-discards %s", mapper))
+	cmd := exec.Command("cryptsetup", "refresh", "--persistent", "--allow-discards", mapper)
```

Comment:
seems like this is needed, to feed the password in stdin in order to modify the device, which makes sense.

Reply:
why wasn't it needed before? Not much changed here, what introduced this need?

Reply:
probably the underlying version of cryptsetup? Maybe we never noticed because this didnt fail the actual encryption, only setting up the discards option so it was not critical as long as it was created properly